Assets/HoloToolkit/UX/Scripts/Distorters/Distorter.cs

```diff
@@ -0,0 +1,102 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+using UnityEngine;
+
+namespace MRTK.UX
```

Comment:
We should probably update the namespace to match the toolkits.

Reply:
How's MRTK.UX -> Holotoolkit.Unity.UX?

Reply:
Correct.